R/new.R

```diff
@@ -20,12 +21,12 @@
 #'
 #' \dontrun{
 #' # All columns must be the same length:
-#' new_tibble(list(a = 1:3, b = 4.6))
+#' new_tibble(list(a = 1:3, b = 4:6))
```

Comment:
This was meant to fail, maybe use `4:5` instead, and perhaps `try()` instead of `\dontrun{}` ?